cl/lib/recap_utils.py

```diff
@@ -120,19 +120,33 @@ def needs_ocr(content):
         "Appellate",
         "Appeal",
         "Case",
+        "Desc",
+        "Main Document",
         "Page",
+        "Received:",
```

Comment:
Similarly, `Received:` wouldn’t be matched by the `bad_starters` check since it’s not at the beginning of the line.

![Screenshot 2025-06-09 at 2 51 10 p m](https://github.com/user-attachments/assets/768793cd-a8ad-4dc7-b9f1-7ff902be358e)

I tested this with the real document, and `needs_ocr` returned `False` for this reason.

In this case, I think it’s better to use a regex pattern to detect it, something like:

```
received_re = re.compile(r"Received: ?\d{2}/\d{2}/\d{2}(?:\d{2})?")
if received_re.search(line):
            continue
```